### PR TITLE
Button: update stories to import from @fluentui/react-components

### PR DESCRIPTION
**Pull request by TristanWatanabe**
### Changes
- updates `react-button` stories to import from `@fluentui/react-components` package suite to demonstrate best practices to users.
- Fixes automatically applied by new `no-restricted-imports` rule.

Part of #23846

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23854/perf-test-react-components/Avatar-mount_base.html">1289</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23854/perf-test-react-components/Avatar-mount.html">1270</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23854/perf-test-react-components/Button-mount_base.html">878</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23854/perf-test-react-components/Button-mount.html">872</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23854/perf-test-react-components/FluentProvider-mount_base.html">1597</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23854/perf-test-react-components/FluentProvider-mount.html">1612</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23854/perf-test-react-components/FluentProviderWithTheme-mount_base.html">557</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23854/perf-test-react-components/FluentProviderWithTheme-mount.html">555</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23854/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">535</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23854/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">526</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23854/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">577</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23854/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">567</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23854/perf-test-react-components/MakeStyles-mount_base.html">1963</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23854/perf-test-react-components/MakeStyles-mount.html">1948</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23854/perf-test-react-components/SpinButton-mount_base.html">2437</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23854/perf-test-react-components/SpinButton-mount.html">2387</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-components/react-button/src/stories/Button/ButtonIcon.stories.tsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
 import { bundleIcon, CalendarMonthFilled, CalendarMonthRegular } from '@fluentui/react-icons';
-import { Tooltip } from '@fluentui/react-tooltip';
-import { Button } from '@fluentui/react-button';
+import { Tooltip, Button } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/Button/ButtonIcon.stories.tsx:3]: _nit:_ Order these alphabetically

```suggestion
import { Button, Tooltip } from '@fluentui/react-components';
```

File: packages/react-components/react-button/src/stories/Button/ButtonSize.stories.tsx [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
 import { bundleIcon, CalendarMonthFilled, CalendarMonthRegular } from '@fluentui/react-icons';
-import { Tooltip } from '@fluentui/react-tooltip';
-import { Button } from '@fluentui/react-button';
+import { Tooltip, Button } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/Button/ButtonSize.stories.tsx:3]: _nit:_ Order these alphabetically

```suggestion
import { Button, Tooltip } from '@fluentui/react-components';
```

File: packages/react-components/react-button/src/stories/CompoundButton/CompoundButtonAsToggleButton.stories.tsx [hunk 3]
```diff
@@ -1,19 +1,24 @@
...
 } from '../../components/ToggleButton/useToggleButtonStyles';
-import type { ForwardRefComponent } from '@fluentui/react-utilities';
-import type { CompoundButtonProps, CompoundButtonState } from '@fluentui/react-button';
-import type { ToggleButtonProps, ToggleButtonState } from '@fluentui/react-button';
+import type {
+  ForwardRefComponent,
+  CompoundButtonProps,
+  CompoundButtonState,
+  ToggleButtonProps,
+  ToggleButtonState,
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/CompoundButton/CompoundButtonAsToggleButton.stories.tsx:20]: _nit:_ Order these alphabetically

```suggestion
  CompoundButtonProps,
  CompoundButtonState,
  ForwardRefComponent,
  ToggleButtonProps,
  ToggleButtonState,
```

File: packages/react-components/react-button/src/stories/CompoundButton/CompoundButtonDefault.stories.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { bundleIcon, CalendarMonthFilled, CalendarMonthRegular } from '@fluentui/react-icons';
-import { CompoundButton, CompoundButtonProps } from '@fluentui/react-button';
+import { CompoundButton, CompoundButtonProps } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/CompoundButton/CompoundButtonDefault.stories.tsx:3]: _nit:_ Separate props into `import type` statement

```suggestion
import { CompoundButton } from '@fluentui/react-components';
import type { CompoundButtonProps } from '@fluentui/react-components';
```

File: packages/react-components/react-button/src/stories/CompoundButton/CompoundButtonIcon.stories.tsx [hunk 5]
```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
 import { bundleIcon, CalendarMonthFilled, CalendarMonthRegular } from '@fluentui/react-icons';
-import { Tooltip } from '@fluentui/react-tooltip';
-import { CompoundButton } from '@fluentui/react-button';
+import { Tooltip, CompoundButton } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/CompoundButton/CompoundButtonIcon.stories.tsx:3]: _nit:_ Order these alphabetically

```suggestion
import { CompoundButton, Tooltip } from '@fluentui/react-components';
```

File: packages/react-components/react-button/src/stories/MenuButton/MenuButtonAppearance.stories.tsx [hunk 6]
```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
-import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-menu';
-import { MenuButton } from '@fluentui/react-button';
+import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, MenuButton } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/MenuButton/MenuButtonAppearance.stories.tsx:2]: _nit:_ Order these alphabetically

```suggestion
import { Menu, MenuButton, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-components';
```

File: packages/react-components/react-button/src/stories/MenuButton/MenuButtonDefault.stories.tsx [hunk 7]
```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
-import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-menu';
-import { MenuButton } from '@fluentui/react-button';
+import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, MenuButton } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/MenuButton/MenuButtonDefault.stories.tsx:2]: _nit:_ Order these alphabetically

```suggestion
import { Menu, MenuButton, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-components';
```

File: packages/react-components/react-button/src/stories/MenuButton/MenuButtonDisabled.stories.tsx [hunk 8]
```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
-import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-menu';
-import { MenuButton } from '@fluentui/react-button';
+import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, MenuButton } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/MenuButton/MenuButtonDisabled.stories.tsx:2]: _nit:_ Order these alphabetically

```suggestion
import { Menu, MenuButton, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-components';
```

File: packages/react-components/react-button/src/stories/MenuButton/MenuButtonIcon.stories.tsx [hunk 9]
```diff
@@ -6,10 +6,16 @@ import {
...
-import { MenuButton } from '@fluentui/react-button';
+import {
+  Menu,
+  MenuItem,
+  MenuList,
+  MenuPopover,
+  MenuTrigger,
+  Tooltip,
+  makeStyles,
+  MenuButton,
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/MenuButton/MenuButtonIcon.stories.tsx:17]: _nit:_ Order these alphabetically

```suggestion
  makeStyles,
  Menu,
  MenuButton,
  MenuItem,
  MenuList,
  MenuPopover,
  MenuTrigger,
  Tooltip,
```

File: packages/react-components/react-button/src/stories/MenuButton/MenuButtonShape.stories.tsx [hunk 10]
```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
-import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-menu';
-import { MenuButton } from '@fluentui/react-button';
+import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, MenuButton } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/MenuButton/MenuButtonShape.stories.tsx:2]: _nit:_ Order these alphabetically

```suggestion
import { Menu, MenuButton, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-components';
```

File: packages/react-components/react-button/src/stories/MenuButton/MenuButtonSize.stories.tsx [hunk 11]
```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
-import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-components';
-import { MenuButton } from '@fluentui/react-button';
+import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, MenuButton } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/MenuButton/MenuButtonSize.stories.tsx:2]: _nit:_ Order these alphabetically

```suggestion
import { Menu, MenuButton, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-components';
```

File: packages/react-components/react-button/src/stories/MenuButton/MenuButtonSizeLarge.stories.tsx [hunk 12]
```diff
@@ -1,8 +1,6 @@
 import * as React from 'react';
 import { bundleIcon, CalendarMonthFilled, CalendarMonthRegular } from '@fluentui/react-icons';
-import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-menu';
-import { Tooltip } from '@fluentui/react-tooltip';
-import { MenuButton } from '@fluentui/react-button';
+import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, Tooltip, MenuButton } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/MenuButton/MenuButtonSizeLarge.stories.tsx:3]: _nit:_ Order these alphabetically

```suggestion
import { Menu, MenuButton, MenuItem, MenuList, MenuPopover, MenuTrigger, Tooltip } from '@fluentui/react-components';
```

File: packages/react-components/react-button/src/stories/MenuButton/MenuButtonSizeMedium.stories.tsx [hunk 13]
```diff
@@ -1,8 +1,6 @@
 import * as React from 'react';
 import { bundleIcon, CalendarMonthFilled, CalendarMonthRegular } from '@fluentui/react-icons';
-import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-menu';
-import { Tooltip } from '@fluentui/react-tooltip';
-import { MenuButton } from '@fluentui/react-button';
+import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, Tooltip, MenuButton } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/MenuButton/MenuButtonSizeMedium.stories.tsx:3]: _nit:_ Order these alphabetically

```suggestion
import { Menu, MenuButton, MenuItem, MenuList, MenuPopover, MenuTrigger, Tooltip } from '@fluentui/react-components';
```

File: packages/react-components/react-button/src/stories/MenuButton/MenuButtonSizeSmall.stories.tsx [hunk 14]
```diff
@@ -1,8 +1,6 @@
 import * as React from 'react';
 import { bundleIcon, CalendarMonthFilled, CalendarMonthRegular } from '@fluentui/react-icons';
-import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-menu';
-import { Tooltip } from '@fluentui/react-tooltip';
-import { MenuButton } from '@fluentui/react-button';
+import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, Tooltip, MenuButton } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/MenuButton/MenuButtonSizeSmall.stories.tsx:3]: _nit:_ Order these alphabetically

```suggestion
import { Menu, MenuButton, MenuItem, MenuList, MenuPopover, MenuTrigger, Tooltip } from '@fluentui/react-components';
```

File: packages/react-components/react-button/src/stories/MenuButton/MenuButtonWithLongText.stories.tsx [hunk 15]
```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
-import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-menu';
-import { MenuButton } from '@fluentui/react-button';
+import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, MenuButton } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/MenuButton/MenuButtonWithLongText.stories.tsx:2]: _nit:_ Order these alphabetically

```suggestion
import { Menu, MenuButton, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-components';
```

File: packages/react-components/react-button/src/stories/SplitButton/SplitButtonIcon.stories.tsx [hunk 16]
```diff
@@ -6,10 +6,17 @@ import {
...
-import { SplitButton, MenuButtonProps } from '@fluentui/react-button';
+import {
+  Menu,
+  MenuItem,
+  MenuList,
+  MenuPopover,
+  MenuTrigger,
+  Tooltip,
+  makeStyles,
+  SplitButton,
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/SplitButton/SplitButtonIcon.stories.tsx:17]: _nit:_ Order these alphabetically

```suggestion
  makeStyles,
  Menu,
  MenuItem,
  MenuList,
  MenuPopover,
  MenuTrigger,
  SplitButton,
  Tooltip,
```

File: packages/react-components/react-button/src/stories/SplitButton/SplitButtonSizeLarge.stories.tsx [hunk 17]
```diff
@@ -1,8 +1,7 @@
 import * as React from 'react';
 import { bundleIcon, CalendarMonthFilled, CalendarMonthRegular } from '@fluentui/react-icons';
-import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-menu';
-import { Tooltip } from '@fluentui/react-tooltip';
-import { SplitButton, MenuButtonProps } from '@fluentui/react-button';
+import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, Tooltip, SplitButton } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/SplitButton/SplitButtonSizeLarge.stories.tsx:3]: _nit:_ Order these alphabetically

```suggestion
import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, SplitButton, Tooltip } from '@fluentui/react-components';
```

File: packages/react-components/react-button/src/stories/SplitButton/SplitButtonSizeMedium.stories.tsx [hunk 18]
```diff
@@ -1,8 +1,7 @@
 import * as React from 'react';
 import { bundleIcon, CalendarMonthFilled, CalendarMonthRegular } from '@fluentui/react-icons';
-import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-menu';
-import { Tooltip } from '@fluentui/react-tooltip';
-import { MenuButtonProps, SplitButton } from '@fluentui/react-button';
+import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, Tooltip, SplitButton } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/SplitButton/SplitButtonSizeMedium.stories.tsx:3]: _nit:_ Order these alphabetically

```suggestion
import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, SplitButton, Tooltip } from '@fluentui/react-components';
```

File: packages/react-components/react-button/src/stories/SplitButton/SplitButtonSizeSmall.stories.tsx [hunk 19]
```diff
@@ -1,8 +1,7 @@
 import * as React from 'react';
 import { bundleIcon, CalendarMonthFilled, CalendarMonthRegular } from '@fluentui/react-icons';
-import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-menu';
-import { Tooltip } from '@fluentui/react-tooltip';
-import { MenuButtonProps, SplitButton } from '@fluentui/react-button';
+import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, Tooltip, SplitButton } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/SplitButton/SplitButtonSizeSmall.stories.tsx:3]: _nit:_ Order these alphabetically

```suggestion
import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, SplitButton, Tooltip } from '@fluentui/react-components';
```

File: packages/react-components/react-button/src/stories/ToggleButton/ToggleButtonIcon.stories.tsx [hunk 20]
```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
 import { bundleIcon, CalendarMonthFilled, CalendarMonthRegular } from '@fluentui/react-icons';
-import { Tooltip } from '@fluentui/react-tooltip';
-import { ToggleButton } from '@fluentui/react-button';
+import { Tooltip, ToggleButton } from '@fluentui/react-components';
```
Comment (khmakoto) [inline packages/react-components/react-button/src/stories/ToggleButton/ToggleButtonIcon.stories.tsx:3]: _nit:_ Order these alphabetically

```suggestion
import { ToggleButton, Tooltip } from '@fluentui/react-components';
```

**Comment by khmakoto** [comment]
Seems like some suggestions were hidden in the big list by GitHub and haven't been applied yet (for some reason GitHub is not allowing me to apply them myself).

**Comment by TristanWatanabe** [comment]
@khmakoto sorry, didnt notice those other suggestions - think I applied all of them now!

**Review by khmakoto — APPROVED** [review]

**Pull request merged**